deeplabcut/pose_estimation_tensorflow/core/evaluate.py

```diff
@@ -297,23 +294,30 @@ def return_evaluate_network_data(
             )
         ),
     )
-    path_test_config = Path(modelfolder) / "test" / "pose_cfg.yaml"
-    # Load meta data
-    data, trainIndices, testIndices, trainFraction = auxiliaryfunctions.load_metadata(
-        os.path.join(cfg["project_path"], metadatafn)
+    path_train_config, path_test_config, _ = deeplabcut.return_train_network_path(
+        config=config,
+        shuffle=shuffle,
+        trainingsetindex=trainingsetindex,
+        modelprefix=modelprefix,
     )
 
     try:
-        dlc_cfg = load_config(str(path_test_config))
+        test_pose_cfg = load_config(str(path_test_config))
     except FileNotFoundError:
         raise FileNotFoundError(
             "It seems the model for shuffle %s and trainFraction %s does not exist."
             % (shuffle, trainFraction)
         )
 
+    train_pose_cfg = load_config(str(path_train_config))
+    # Load meta data
+    data, trainIndices, testIndices, _ = auxiliaryfunctions.load_metadata(
+        Path(cfg["project_path"]) / train_pose_cfg["metadataset"],
```

Comment:
I haven't tested it but is passing a `Path` instead of a `str` to `load_metadata()` fine?